src/beautifiers/terraformfmt.coffee

```diff
@@ -0,0 +1,24 @@
+###
+Requires terraform installed
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Terraformfmt extends Beautifier
+  name: "terraformfmt"
+  link: "https://www.terraform.io/docs/commands/fmt.html"
+  isPreInstalled: false
```

Comment:
New beautifiers should use `executables` feature: https://github.com/Glavin001/atom-beautify/issues/1687

Here's one Pull Request for example: https://github.com/Glavin001/atom-beautify/pull/1726/files#diff-4bf70ccc1cd2e9c4d2706b85b61fb41c

Please let me know if you have any questions. The Executables feature needs more documentation, so currently all we have are existing beautifiers are examples. See https://github.com/Glavin001/atom-beautify#beautifiers

Reply:
If I change this, the exec is not found anymore. Testing with where (on windows) works.
Any Idea?

Reply:
@Glavin001 now everything should be fixed.